src/components/itemMediaInfo/itemMediaInfo.js

```diff
@@ -137,7 +137,7 @@ const attributeDelimiterHtml = layoutManager.tv ? '' : '<span class="hide">: </s
             if (stream.Channels) {
                 attributes.push(createAttribute(globalize.translate('MediaInfoChannels'), `${stream.Channels} ch`));
             }
-            if (stream.BitRate && stream.Codec !== 'mjpeg') {
+            if (stream.BitRate && (stream.Type === 'Video' || stream.Type === 'Audio')) {
```

Comment:
`BitRate: undefined` for the embedded cover image (`mjpeg`), so I think even the old check is unnecessary.

Reply:
Removed.